### DISCUSSION TITLE
how to change component's default colorPalette in v3

### DISCUSSION THREAD

**Discussion by xiefenga** [chakra-ui/chakra-ui#9009] [Q&A]
chakra ui v3 default colorPalette props value is gray，i want to change it to adapt to the original color of the system.

for example change it to blue, i tried custom recipe but not work

```js

const buttonRecipe = defineRecipe({
  defaultVariants: {
    colorPalette: 'blue',
  },
})

export const system = createSystem(defaultConfig, {
  disableLayers: true,
  theme: {
    recipes: {
      button: buttonRecipe,
    },
  },
})
```

**Comment by y-wertz-priomold** [comment]
Modify your provider (here I used custom palette "priomold" and default dark mode):

```

import { ChakraProvider, Theme } from "@chakra-ui/react";
import { ColorModeProvider } from "./ui/color-mode";
import { system } from "@/theme/theme";

export function CustomChakraProvider(props: React.PropsWithChildren) {
  return (
    <ChakraProvider value={system}>
      <ColorModeProvider forcedTheme="dark">
        <Theme colorPalette="priomold">{props.children}</Theme>
      </ColorModeProvider>
    </ChakraProvider>
  );
}

```

**Comment by lucasishuman** [comment]
https://www.chakra-ui.com/guides/theming-change-default-color-palette

note - i had to use `globalCss` instead of `globalStyles`, like...

```js
const appConfig = defineConfig({
  globalCss : {
    html: {
      colorPalette: 'blue',
    },
  },
});
```

**Reply by bertho-zero** [reply]
Is there no way to change the default `colorPalette` per recipe/component?

**Comment by auaa-x** [comment]
> Is there no way to change the default colorPalette per recipe/component?
@bertho-zero 

According to [the official doc](https://www.chakra-ui.com/docs/styling/virtual-color#recipes), it seems colorPalette should be defined in `bsae` instead of `defaultVariants`, and I got mine working after this modification.
<img width="813" alt="image" src="https://github.com/user-attachments/assets/93c9c84c-4517-4b89-969f-eb76e6c6f094" />
